pep-0205.txt

```diff
@@ -109,7 +109,7 @@ An alternate approach to invalidation is that the de-allocation
 code to be aware of the possibility of weak references and make a
 specific call into the weak-reference management code to all
 invalidation whenever an object is deallocated.  This requires a
-change in the tp_dealloc handler for weakly-referencable objects;
+change in the tp_dealloc handler for weakly-referenceable objects;
```

Comment:
This is probably fine, we can add it to `.codespell/ignore-words.txt`:

https://en.wiktionary.org/wiki/referencable

Reply:
Thank you, reverted this line and added the exception.